recursio-client/UI/HUD.gd

```diff
@@ -78,17 +81,25 @@ func update_special_movement_ammo(amount: int) -> void:
 	_dash.text = str(amount)
 
 
-func update_weapon_type(img_bullet, color) ->void:
+func update_weapon_type(img_bullet, color) -> void:
 	Logger.info("Update ammo type", "HUD")
 	_ammo_type_bg.modulate = color
 	_ammo_type.texture = img_bullet
 
 
+func activate_spawn_point(timeline_index) -> void:
+	Logger.info("Activate SpawnPoint " + str(timeline_index), "HUD")
+	for spawn_point in _spawn_points:
+		if spawn_point.get_child_count() > 0:
+			spawn_point.get_child(0).set_active(spawn_point.get_index() == timeline_index)
```

Comment:
is there another way to do this? get_child(0) can be a bit volatile... or is there a reason for it definitely always being the first child?

Reply:
Good point! Although it is definitely always the first and only child, I thinks it is clearer for the reader to get the node per name 👍